packages/edge-login-ui-rn/src/common/actions/LoginAction.js

```diff
@@ -126,7 +126,7 @@ export function userLoginWithTouchId (data: Object) {
       context,
       data.username,
       'Touch to login user: `' + data.username + '`',
-      null,
+      'Login with password',
```

Comment:
I'm not sure where this message gets used. It seems to be `localizedFallbackTitle` from [`LAPolicyDeviceOwnerAuthenticationWithBiometrics`](https://developer.apple.com/documentation/localauthentication/lapolicy/lapolicydeviceownerauthenticationwithbiometrics), but I don't have a Apple device to test with.